### PR TITLE
Updated Documentation with Guideline

### PR DESCRIPTION
**Pull request by testerslol123**
Refer to this issue #16 

I just updated those guideline with how to operate this code in your local environment
- Guideline in how to operate project
- Updated documentation credential user to login system

### CODE REVIEW

**Comment by testerslol123** [comment]
How to add label hacktober fest to this?
It was my first PR, so I want to make sure it available.

@Tjoosten Thank you

**Comment by Tjoosten** [comment]
Hey @testerslol123  

Looks good! Thx for your effort that u put in this one. Wll merge this after my lunch break if you don't mind. 

Kind regards, 
Tim Joosten

File: README.md [hunk 1]
```diff
@@ -14,6 +14,34 @@ The boilerplate contains the following features:
...
+Here is how to use this project step by step:
+- After you clone this repo, don't forget to do (Make sure you install composer first, refer [here](https://getcomposer.org)):
+```composer install``
+- Setup your environment, copy the .env.example to .env, and filling those credential database.
+- After you do the install composer, do the following ```php artisan key:generate```
+- Then do migrate and seeding the database with ```php artisan migrate:refresh --seed```
+- After you done those steps, time to serving the laravel by using ```php artisan serve```
+- Then go to your browser (by default it would using port 8000), so you can access with ```localhost:8000/home```
+
+## Login Credential
```
Comment (Tjoosten) [inline README.md:31]: Maybe it is a better wording choice if we replace `Login Credential` to `Default login credentials`

Comment (Tjoosten) [reply README.md:31]: And also u forked from the main branch. Which didn't include the logo that is added in the develop branch

Comment (testerslol123) [reply README.md:31]: Okay, I would fix it. And I just saw there are 2 branch, main and develop. 
Thank you for raising those issue.

**Comment by Tjoosten** [comment]
Closing this one because #18

**Pull request closed without merging**